ads/medyanet.js

```diff
@@ -23,10 +23,11 @@ import {validateData} from '../3p/3p';
  */
 
 export function medyanet(global, data) {
-  validateData(data, ['slot']);
+  validateData(data, ['slot', 'domain']);
```

Comment:
Do you have any optional arguments that you would like to validate here?

Reply:
yes, slot and domain parameters are required.

Reply:
Sorry if I wasn't clear. I understand that these are the required arguments. You can also add optional parameters as a third argument if you wish. It is optional, so if you are ready I will go ahead and merge.